app/src/main/java/com/sun/ui/sample/activity/SampleActivity.kt

```diff
@@ -17,28 +22,69 @@ import kotlinx.android.synthetic.main.activity_sample.*
  * Show list post got from URL_END_POINT = https://jsonplaceholder.typicode.com/
  */
 @AndroidEntryPoint
-class SampleActivity : BaseActivity<ActivitySampleBinding, SampleViewModel>() {
+class SampleActivity
+    : BaseActivity<ActivitySampleBinding, SampleViewModel>(), LoadStateListener, RepoListener {
```

Comment:
For classes with a long supertype list, put a line break after the colon and align all supertype names horizontally:
https://kotlinlang.org/docs/coding-conventions.html#class-headers